general/session.md

```diff
@@ -0,0 +1,26 @@
+# Session API
```

Comment:
Can we lead off here with a link to the API, and a statement of what it is actually for?

---

The Session API provides a mechanism for associating persisted key/value pairs with a user as they navigate between pages.

It consists of two methods exported from `frontend-js-web`:

- [`getSessionValue(key)`](https://github.com/liferay/liferay-portal/blob/2d2e8ae74e9e7b33a07328277da64f54db576a04/modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/util/session.es.js#L38-L63)
- [`setSessionValue(key, Value)`](https://github.com/liferay/liferay-portal/blob/2d2e8ae74e9e7b33a07328277da64f54db576a04/modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/util/session.es.js#L65-L85)